libs/elasticsearch/tests/integration_tests/_sync/test_chat_history.py

```diff
@@ -51,11 +52,15 @@ def test_memory_with_message_store(
         )
 
         # add some messages
-        memory.chat_memory.add_ai_message("This is me, the AI")
-        memory.chat_memory.add_user_message("This is me, the human")
+        memory.chat_memory.add_messages(
+            [
+                AIMessage("This is me, the AI"),
+                HumanMessage("This is me, the human"),
+            ]
+        )
 
         # get the message history from the memory store and turn it into a json
-        messages = memory.chat_memory.messages
+        messages = memory.chat_memory.get_messages()  # type: ignore[attr-defined]
```

Comment:
Have you considered turning `chat_memory.get_messages()` into `chat_memory.messages` as part of the unasync sed calls?

Reply:
I guess we could, but that would be the only thing that needs `sed`, as everything else is handled natively by unasync. If this was the only problem that required creating a subclass then I think it would make sense, but there's also the sync abstract methods that have to be inserted, so given that we have to use the subclassing trick anyway I'm not sure it makes sense to complicate the build with an additional `sed` step.

Reply:
I'm not sure I'm completely following the mentions of subclassing, plus we're already calling sed as far as I can tell. Anyway, my concern is that the `messages` property is no longer tested. It's not a big concern, and maybe it does not justify using sed or adding a new test file outside of unasync to test it.

Reply:
By subclassing I mean the method that I'm using to re-introduce the missing `messages` property on the sync side, and the abstract sync methods on the async side.

I was wrong, we are using `sed`. I thought I removed those calls, but now that I look at them I think I can remove all but one. The one that changes the `@pytest.mark.asyncio` to `@pytest.mark.sync` has to stay. I believe the other ones are not used, I just copy/pasted them from another project.

But your point is valid. I will change `get_messages()` to `messages`, I have no more arguments against it.

Reply:
Happy to do that as a follow-up too.